src/pages/sighting/statusCard/StatusCard.jsx

```diff
@@ -111,8 +100,11 @@ export default function StatusCard({ sightingData }) {
     detectionSkippedLabelId = 'DETECTION_SKIPPED_NO_MODEL_MESSAGE';
   }
 
+  const formattedDetectionStartTime = detectionStartTime
```

Comment:
Probably an opportunity to DRY this up with formattedPreparationStartTime, but IMO not worth it. I imagine you probably came to the same conclusion lol.

Reply:
Done in https://github.com/WildMeOrg/codex-frontend/pull/432/commits/fcdcadc242572a99ba389041e75abf9dc92eaeaa